### DISCUSSION TITLE
Nuget Content folder has all the CefSharp depenencies.

### DISCUSSION THREAD

**Discussion by GitNewbieHub** [cefsharp/CefSharp#4616] [General]
We recently upgraded CefSharp from 49.0.1 to 117.2.40 in one of our project and noticed the project went from 105KB to 230MB in size when we packaged it into NuGet. We checked the NUPKG file with NugetPackageExplroer and noticed there is Content folder with all the CefSharp dependencies. We didn't change our project files. I then tested with other versions of CefSharp to see if they have the Content folder. Looks like from 88.2.90 ,all the newer versions have content folder with all the dependencies and all the versions prior 88.2.90 don't have the Content folder. I didn't find anything in the release note. Does anyone know if this is intended? How can I build Nuget without the Content file? We want to minimize the NuGet package size.

Also, we noticed there are warning messages like this:

C:\Users\build_made\.nuget\packages\nuget.build.tasks.pack\5.4.0\build\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5100: The assembly 'content\CefSharp.BrowserSubprocess.Core.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project. Move it into the 'lib' folder if it needs to be referenced.

I didn't see the warning messages with old versions also, only 88.2.90 and newer versions.

**Comment by amaitland** [comment]
The `Nuget` packages were restructured as part of https://github.com/cefsharp/CefSharp/issues/3319

See https://github.com/cefsharp/CefSharp/blob/cefsharp/118/NuGet/CefSharp.Common.targets#L99

Add the following to the first property group of your `proj` file

```
<CefSharpBuildAction>None</CefSharpBuildAction>
```

**Reply by GitNewbieHub** [reply]
Thank you!!!
